### PR TITLE
Update CODEOWNERS to @mozilla-services/autograph-devs

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,8 +1 @@
-*       @g-k, @kkleemola, @pwnbus
-
-# the AMO team should review changes that impact Addon signing
-signer/xpi/	@willdurand
```
Comment (willdurand) [inline .github/CODEOWNERS:4]: If possible, I'd like to be pinged explicitly when there are changes related to add-ons.

Comment (jmhodges) [reply .github/CODEOWNERS:4]: oh! Is there an add-ons reviewer team?

Comment (willdurand) [reply .github/CODEOWNERS:4]: Can you try https://github.com/orgs/mozilla-services/teams/addons-engineering please?

Comment (jbuck) [reply .github/CODEOWNERS:4]: @willdurand thank you for the team suggestion, I updated the PR to use that

**Pull request closed without merging**